applicationset/generators/matrix.go

```diff
@@ -44,16 +44,15 @@ func (m *MatrixGenerator) GenerateParams(appSetGenerator *argoprojiov1alpha1.App
 
 	res := []map[string]string{}
 
-	g0, err := m.getParams(appSetGenerator.Matrix.Generators[0], appSet)
+	g0, err := m.getParams(appSetGenerator.Matrix.Generators[0], appSet, map[string]string{})
```

Comment:
nit: I think you can just pass `nil` for the map.

Reply:
Resolved!